frontend/components/Header.vue

```diff
@@ -3,10 +3,11 @@
     <b-container class="px-sm-3">
       <b-navbar-brand>
         <nuxt-link to="/" class="navbar-brand" title="Reef block explorer">
-          <img class="logo" src="/img/polkastats_logo_dark@2x.png" />
+          <img class="logo" src="/img/cere-logo.png" />
+          Cerestats
         </nuxt-link>
       </b-navbar-brand>
-      <a
+      <!-- <a
```

Comment:
Let's remove instead of commenting. We can always revert if it will be needed. Here and in all other places.

Reply:
okay.

Reply:
Removed unused code.